website/description_issues.qmd

```diff
@@ -40,3 +43,38 @@ Document all non-obvious acronyms in the [cran-comments.md](https://usethis.r-li
 ### Details
 
 Most acronyms that are not widely known should be explained. For example, you don’t have to explain *OLS*, *SEO*, or *DNA* but explanations should be provided for: *MEFM* or *OCCDS*. If you are unsure of the acronyms common knowledge, please document all acronyms in the package.
+
+
+---
+
+
+# LICENSE files
+
+## Problem
+
+A file is referenced in the DESCRIPTION field for License when it's not required or the full license has been copied and submitted in the LICENSE file.
+
+## Solution
+
+Remove the reference `+ file LICENSE` and delete the LICENSE file.
+
+### Details
+
+Most licenses don’t need an additional file as these are part of R. The list of which licenses that do require an additional file can be accessed within _R_ by searching for **Note: this is a template, needs + file LICENSE**:
+
+```r
+path_licenses <- RShowDoc("license.db")
+licenses_doc <- as.data.frame(read.dcf(path_licenses))
+```
+
+If an additional file is needed, it should only specify the author and year. 
```

Comment:
This is not entirely true. Authors and year are needed for the MIT license. The BSD file license should contain three lines:

> YEAR:
> COPYRIGHT HOLDER:
> ORGANIZATION:

Reply:
i'll update this!